### PR TITLE
Updates to Ember > 2.16

### PR DESCRIPTION
**Pull request by tylerturdenpants**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -13,21 +13,15 @@ insert_final_newline = true
...
 indent_style = space
 indent_size = 2
 
-[*.css]
+[*.{diff,md}]
 indent_style = space
 indent_size = 2
-
-[*.html]
 indent_style = space
```
Comment (crazymykl) [inline .editorconfig]: This is redundant.

File: .editorconfig [hunk 2]
```diff
@@ -13,21 +13,15 @@ insert_final_newline = true
...
-indent_style = space
-indent_size = 2
-
 [*.hbs]
+insert_final_newline = false
 indent_style = space
 indent_size = 2
 
-[*.css]
+[*.{diff,md}]
```
Comment (crazymykl) [inline .editorconfig]: `md` is already handled below.

**Comment by tylerturdenpants** [comment]
@crazymykl I still need to work on the jquery event deprecation. I will update you when that is finished.

File: .editorconfig (hunk 2 above)
Comment (tylerturdenpants) [reply .editorconfig]: will fix

File: .editorconfig (hunk 1 above)
Comment (tylerturdenpants) [reply .editorconfig]: Will fix

**Comment by tylerturdenpants** [comment]
I think I fixed the .editorconfig.  Let me know what you think.

**Comment by tylerturdenpants** [comment]
The dummy works and passes, but I have found some issues when consuming this from a host app.  Looking into it.

**Comment by tylerturdenpants** [comment]
@crazymykl is there anything else that would keep this from being merged?  Let me know you comments

**Comment by crazymykl** [comment]
No, it looks good to me.

**Comment by tylerturdenpants** [comment]
@crazymykl I might follow this up with a PR that removes ember-auto-import dep because I think its responsible for dep issues in some host app setups (for example, my app).  Can you refrain from cutting a release until I get a chance to confirm?

**Comment by crazymykl** [comment]
Nope, it's already happened, but I'd be happy to put out a 0.4.1.

**Comment by tylerturdenpants** [comment]
ok, thanks!

**Comment by tylerturdenpants** [comment]
PR #13 should be good to go as a 0.4.1 release!

**Pull request merged**